react/SlideToggle/SlideToggle.less

```diff
@@ -50,7 +51,8 @@
 
 .label {
   padding: 0;
-  display: inline;
+  line-height: @grid-row-height * 3;
+  transform: none;
```

Comment:
Ha, that's funny, I literally just finished chatting with @dborysiewicz about exactly this. He's planning to propose a change to the text mixins and components, allowing you to write `<Text baseline={false}>`, which I'm guessing will do the trick here?